sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationTests.cs

```diff
@@ -358,7 +358,6 @@ public void AnalyzeOperationWithErrorTest()
             {
                 ExtractKeyPhrasesActions = new List<ExtractKeyPhrasesAction>()
                 {
-                    new ExtractKeyPhrasesAction(),
```

Comment:
Had to remove dup action type as it is no longer supported. The purpose of the tests still maintains which is to validate that we get an error when setting an invalid version.
It required re-record